tools/plugins/network/src/test/kotlin/net/corda/cli/plugins/network/MemberLookupTest.kt

```diff
@@ -0,0 +1,215 @@
+package net.corda.cli.plugins.network
+
+import com.fasterxml.jackson.databind.JsonNode
+import net.corda.cli.plugins.network.output.Output
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+import net.corda.e2etest.utilities.DEFAULT_CLUSTER
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.databind.node.JsonNodeFactory
+import net.corda.cli.plugins.network.enums.MemberStatus.ACTIVE
+import net.corda.cli.plugins.network.enums.MemberStatus.SUSPENDED
+import net.corda.cli.plugins.network.utils.HoldingIdentityUtils
+import net.corda.membership.lib.MemberInfoExtension
+import org.junit.jupiter.api.BeforeAll
+import picocli.CommandLine
+
+class MemberLookupTest {
+
+    companion object {
+        private lateinit var memberLookup: MemberLookup
+        private lateinit var onboardMgm: OnboardMgm
+        private lateinit var outputStub: OutputStub
+
+        private val CLI_PARAMS = arrayOf(
+            "--target=${DEFAULT_CLUSTER.rest.uri}",
+            "--user=${DEFAULT_CLUSTER.rest.user}",
+            "--password=${DEFAULT_CLUSTER.rest.password}",
+            "--insecure=${true}"
+        )
+
+        private val X500 = arrayOf(
```

Comment:
I think this would align better with naming conventions
```suggestion
        private val x500 = arrayOf(
```